commands/accounts/set_password.go

```diff
@@ -52,17 +53,51 @@ func accountsSetPasswordPreRun(cmd *cobra.Command, _ []string) { //nolint:revive
 	// Copy global flags to config
 	accountsSetPasswordCfg.Pretty = !(cmd.Flag(cli.FlagVerbose).Changed || cmd.Flag(cli.FlagJSON).Changed)
 	accountsSetPasswordCfg.KubeconfigPath = cmd.Flag(cli.FlagKubeconfig).Value.String()
+
+	// Check username
+	if accountsSetPasswordOpts.Username != "" {
+		// Validate provided username for new account.
```

Comment:
here and 3 others below - misleading comment about new account

Reply:
done